### ISSUE TITLE
change default NVDA voice and variant

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5272]
**Reported by joshknnd1982 on 2015-08-06 17:34**
Due to the fact that so many people like eloquence, and also the fact that they cannot use sapi5 eloquence right away when NVDA is first started... or some people cannot afford eloquence... Therefore I would like to request the default voice in NVDA please change it to the latest speechPlayer in ESpeak with Edward as the default variant set to US english. I would also ask that the volume of Edward be slightly increased.

**Comment by bhavyashah** [comment]
@jcsteh @michaelDCurran Could you please briefly address this request and subsequently close this ticket?

**Comment by leonardder** [comment]
@MichaelDCurran, do you think making Edward the default for Espeak instead of Max is a good idea?

Related to #8177 and #8176

**Comment by michaelDCurran** [comment]
Many english speaking users would be very happy with this change if it 
did happen. However, I fear for non-english speakers, Edward may not be 
as good as Max as  there may still be some issues with the way eSpeak 
maps particular phonemes to Klatt.

**Comment by Adriani90** [comment]
This affects only Windows 8.1 users, since in Windows 10 and higher the default voice is one core.
cc: @gerald-hartig maybe you can decide whether this needs attention or should we close it as not planned? I guess the user base of Windows 8.1 is very very low nowadays.

**Comment by gerald-hartig** [comment]
If this only affects windows 8.1 users, then I think given that (A) Windows 8.1 reached end of life in Jan 2023, (B) less than 2% of our users are running windows 8.1, and (C) there's been very little activity on this issue in the 9 years since it was raised, we can close this issue as stale.
